--- a/Mathlib/LinearAlgebra/LinearIndependent/Defs.lean
+++ b/Mathlib/LinearAlgebra/LinearIndependent/Defs.lean
@@ -889,7 +889,7 @@
 lemma LinearIndepOn.singleton' (hi : ∀ r : R, r • v i = 0 → r = 0) : LinearIndepOn R v {i} :=
   LinearIndependent.of_subsingleton' ⟨i, rfl⟩ hi
 
-variable [IsDomain R] [IsTorsionFree R M]
+variable [NoZeroDivisors R] [IsTorsionFree R M]
 
 lemma LinearIndependent.of_subsingleton [Subsingleton ι] (i : ι) (hi : v i ≠ 0) :
     LinearIndependent R v := .of_subsingleton' i (by simp [hi])
