--- a/Mathlib/Algebra/Algebra/Tower.lean
+++ b/Mathlib/Algebra/Algebra/Tower.lean
@@ -448,7 +448,7 @@
 namespace Algebra
 
 variable [CommSemiring R] [Semiring A] [IsDomain A] [Semiring B] [Algebra R A] [Algebra R B]
-variable [AddCommGroup M] [Module R M] [Module A M] [Module B M]
+variable [AddCommMonoid M] [Module R M] [Module A M] [Module B M]
 variable [IsScalarTower R A M] [IsScalarTower R B M] [SMulCommClass A B M]
 
 theorem lsmul_injective [Module.IsTorsionFree A M] {x : A} (hx : x ≠ 0) :
