--- a/Mathlib/Analysis/Normed/Group/CocompactMap.lean
+++ b/Mathlib/Analysis/Normed/Group/CocompactMap.lean
@@ -27,7 +27,7 @@
 open Filter Metric
 
 variable {𝕜 E F 𝓕 : Type*}
-variable [NormedAddCommGroup E] [NormedAddCommGroup F]
+variable [SeminormedAddCommGroup E] [NormedAddCommGroup F]
 variable {f : 𝓕}
 
 theorem CocompactMapClass.norm_le [ProperSpace F] [FunLike 𝓕 E F] [CocompactMapClass 𝓕 E F]
