--- a/Mathlib/Algebra/Group/Submonoid/Pointwise.lean
+++ b/Mathlib/Algebra/Group/Submonoid/Pointwise.lean
@@ -289,7 +289,7 @@
 
 namespace Set.IsPWO
 
-variable [CommMonoid α] [PartialOrder α] [IsOrderedCancelMonoid α] {s : Set α}
+variable [CommMonoid α] [Preorder α] [IsOrderedCancelMonoid α] {s : Set α}
 
 @[to_additive]
 theorem submonoid_closure (hpos : ∀ x : α, x ∈ s → 1 ≤ x) (h : s.IsPWO) :
